src/Sylius/Bundle/AdminBundle/Resources/views/Order/Show/Summary/_item.html.twig

```diff
@@ -23,7 +25,7 @@
         {{ money.format(item.units.first.adjustmentsTotal(unitPromotionAdjustment), order.currencyCode) }}
     </td>
     <td class="right aligned unit-order-discount">
-        <span style="font-style: italic;">{{ money.format(item.units.first.adjustmentsTotal(orderPromotionAdjustment), order.currencyCode) }}</span>
+        <span style="font-style: italic;">~{{ money.format(item.units.first.adjustmentsTotal(orderPromotionAdjustment), order.currencyCode) }}</span>
```

Comment:
This value is already displayed with `font-style: italic;` and I guess that adding `~` should be reverted

Reply:
I talked to @CoderMaggie - she thinks its a nice way to signify the variance this specific value might have. You can see [here](https://github.com/Sylius/Sylius/issues/11112#issuecomment-584628423), why. I can fix the tests by adding the ~ to the `ShowPage` so it gets trimmed away nicely.

Or I can remove it - your call, I am not opposed to any solution :slightly_smiling_face: